### ISSUE TITLE
BUG: Alignment

### ISSUE THREAD

**Issue by larsoner** [mne-tools/fiff-constants#11]
Files should be aligned using spaces to ensure uniformity across readers. So far #8 will only take care of `DictionaryTypes.txt`.

**Comment by mkajola** [comment]
Yes, i agree about spaces.In general, this can be done later as you suggested.

**Comment by larsoner** [comment]
Sounds good. I'm going to leave this open as a reminder for us to do it at some point. A nice Sunday hack for when someone is bored and/or procrastinating :)
